department-of-veterans-affairs/caseflow
# Fix Location filter in Assign Hearings

Resolves #9528 

### Description
Misspelled function prevented filter icon to react to click event.

### Acceptance Criteria

### Testing Plan
1. Go to Assign Hearings
2. Click on Location filter icon
3. Confirm dropdown displays with all available locations in the table.
